src/basic/fd-util.c

```diff
@@ -913,6 +913,36 @@ int fd_is_opath(int fd) {
         return FLAGS_SET(r, O_PATH);
 }
 
+int fd_extrinsic_flags_safe(int fd) {
```

Comment:
hmm, maybe call this fd_has_regular_flags()?

i mean, that's what this is about after all, that the flags are "regular", and not "weird" in any case.

Reply:
or fd_verify_regular_flags()

Reply:
or fd_verify_safe_flags()